src/Core/src/Dispatching/Dispatcher.Windows.cs

```diff
@@ -79,8 +78,6 @@ public void Stop()
 			if (!IsRunning)
 				return;
 
-			_timer.Tick -= OnTimerTick;
```

Comment:
Perhaps this should be in Dispose ?   I don't see it getting unregistered so it looks like a potential leak.

Reply:
Yes. The new fix has issues with retention. I thought about this a bit more.

- Originally the way to unregister was to call `Stop()` and it feels more intuitive, especially for repeatable timers, which I believe are more common, since that is the default.
- The android counterpart effectively unregisters on `Stop()`. It would be nice if behaviors were similar.
- `IDispatcherTimer` is not disposable right now. We can make it disposable, but it seems heavier and less intuitive solution than auto-unregistering on Stop().
- Unregistering in `Dispose()` would open a scenario where someone calls `Start()` after disposing and we would need to handle that in some predictable way. Other properties may need to check/throw as well, for consistency.
- Overall, calling 'Start()` after `Stop()` does not seem to be a key scenario. We do need to have some rational behavior, but not necessarily optimize for it.

Considering all the above, I think the original solution in this PR may be more acceptable after all.
I.E. we would unregister in `Stop()` and get a new timer if `Start()` is happened to be called again.

Reply:
Now I think I am getting confused. The existing code is to unregister in stop and register in start. Not sure what the bug was actually doing because a double start is a no-op and a stop-start should have unregistered and re-registered.